core/src/test/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionFileSystemOutputTest.java

```diff
@@ -21,23 +21,21 @@
 
 package com.microsoft.applicationinsights.internal.channel.common;
 
-import java.io.File;
-import java.io.IOException;
-import java.util.Collection;
-
-import com.microsoft.applicationinsights.internal.util.LocalFileSystemUtils;
-import org.junit.Ignore;
-import org.junit.Test;
-
-import org.apache.commons.io.FileUtils;
-
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 
+import com.microsoft.applicationinsights.internal.util.LocalFileSystemUtils;
+import java.io.File;
+import java.util.Collection;
+import org.apache.commons.io.FileUtils;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
 public final class TransmissionFileSystemOutputTest {
     private final static String TRANSMISSION_FILE_EXTENSION = "trn";
-    private final static int SIZE_OF_TRANSMISSION_CONTENT = 100;
+    private final static int SIZE_OF_TRANSMISSION_CONTENT = 349525;
```

Comment:
Please show where this number came from, i.e. `100 * 1024 *1024; // 100 KB`. I'm not really sure what this number is.